hypha/apply/funds/views_all.py

```diff
@@ -355,11 +366,47 @@ def bulk_update_submissions_status(request: HttpRequest) -> HttpResponse:
 
     submissions = ApplicationSubmission.objects.filter(id__in=submission_ids)
 
-    redirect: HttpResponse = BatchDeterminationCreateView.should_redirect(
-        request, submissions, transitions
-    )
-    if redirect:
-        return HttpResponseClientRedirect(redirect.url)
+    # should redirect
```

Comment:
Moved `should_redirect` method directly in here to avoid raising an Error, instead showing the error to the user without performing the batch action. 

It is hard to find all the cases at once so I guess it is a better way to avoid 500. We can include the cases to validation as soon as it is reported.